apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -22,8 +134,15 @@
   padding: 20px;
 }
 
+.webcam-container {
+  background-color: white;
+  margin-bottom: 10px;
+}
+
 .webcam {
-  width: 200px;
+  width: 90%;
+  margin: 0 auto;
```

Comment:
Properties should be ordered width, padding, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)